### ISSUE TITLE
port forwarding does not respect SKAFFOLD_NAMESPACE

### ISSUE THREAD

**Issue by balopat** [GoogleContainerTools/skaffold#1177]
Repro: 

1.  `SKAFFOLD_NAMESPACE` set to a non default namespace 
2. run the microservices example with `skaffold dev` 

````
Port Forwarding leeroy-app-57ffdf98b9-7cp8d 50051 -> 50051
WARN[0009] port forwarding pod failed: port forwarding: port forwarding pod: leeroy-app-57ffdf98b9-7cp8d, port: 50051, err: Error from server (NotFound): pods "leeroy-app-57ffdf98b9-7cp8d" not found
: exit status 1 
Port Forwarding leeroy-web-85975c7955-c9hcg 8080 -> 8080
[leeroy-app-57ffdf98b9-7cp8d leeroy-app] 2018/10/18 18:35:56 leeroy app server ready
WARN[0010] port forwarding pod failed: port forwarding: port forwarding pod: leeroy-web-85975c7955-c9hcg, port: 8080, err: Error from server (NotFound): pods "leeroy-web-85975c7955-c9hcg" not found
: exit status 1 
[leeroy-web-85975c7955-c9hcg leeroy-web] 2018/10/18 18:35:56 leeroy web server ready
````

**Comment by eaceaser** [comment]
Ran into a similar manifestation of this issue, where the namespace is being set in a helm chart release -- this isn't threaded through into the port-forward command either.

**Comment by dgageot** [comment]
I'm going to close this one. I can't reproduce it and it seems it was fixed by c55e88a7a24e57cbdb061bc6f0fe4810d3c7235a
